### ISSUE TITLE
Example usage

### ISSUE THREAD

**Issue by samuelstevens** [samuelstevens/vcell#13]
Inside an `experiments/` file, use the dataloader/dataset class to calculate the validation loss for an untrained and a trained model so that others can copy the usage.
